packages/notifications/bot.py

```
@@ -0,0 +1,1453 @@
+    try:
+        data = await _proxy_get("/admin/api/status")
+        services = data.get("services", {})
+        tunnel_url = data.get("tunnel_url", "unknown")
+        lines = ["*Service status:*"]
+        for svc, info in services.items():
+            state = "running" if info.get("running") else "stopped"
+            icon = "🟢" if info.get("running") else "🔴"
+            lines.append(f"  {icon} {svc}: {state}")
+        lines.append(f"\nTunnel: `{tunnel_url}`")
+        return "\n".join(lines)
+    except Exception as exc:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)